logger.go

```diff
@@ -221,6 +221,12 @@ type LoggerOptions struct {
 	// Color the output. On Windows, colored logs are only avaiable for io.Writers that
 	// are concretely instances of *os.File.
 	Color ColorOption
+
+	// A filter which is called with the log information and returns true if the value
+	// should not be logged (meaning return true it ignore the log entry).
```

Comment:
The phrase in parentheses reads like a typo, but I'm not sure how to correct it.

Reply:
I've make the sentence better and removed the parenthetical.